engine/app/controllers/good_job/base_controller.rb

```diff
@@ -24,10 +24,28 @@ class BaseController < ActionController::Base # rubocop:disable Rails/Applicatio
       request.content_security_policy_nonce_generator = ->(_request) { SecureRandom.base64(16) }
     end
 
+    def default_url_options(options = {})
```

Comment:
Is there a particular need for it to be public? Now it's treated as an action.

Reply:
@igas good catch. Could you make a PR for that?